File: app/src/main/java/org/jay/launchstarterexample/tasks/InitFrescoTask.java
package org.jay.launchstarterexample.tasks;

import com.facebook.drawee.backends.pipeline.Fresco;
import com.facebook.imagepipeline.core.ImagePipelineConfig;
import com.facebook.imagepipeline.listener.RequestListener;
import org.jay.launchstarter.Task;
import org.jay.launchstarterexample.net.FrescoTraceListener;

import java.util.HashSet;
import java.util.Set;

public class InitFrescoTask extends Task {

    @Override
    public void run() {
        Set<RequestListener> listenerset = new HashSet<>();
        listenerset.add(new FrescoTraceListener());
        ImagePipelineConfig config = ImagePipelineConfig.newBuilder(mContext).setRequestListeners(listenerset)
                .build();
        Fresco.initialize(mContext,config);
    }
}

File: app/src/main/java/org/jay/appstarterexample/tasks/InitFrescoTask.kt
package org.jay.appstarterexample.tasks

import com.facebook.imagepipeline.listener.RequestListener
import org.jay.appstarterexample.net.FrescoTraceListener
import com.facebook.imagepipeline.core.ImagePipelineConfig
import com.facebook.drawee.backends.pipeline.Fresco
import org.jay.appstarter.Task
import java.util.HashSet

class InitFrescoTask : Task() {
    override fun run() {
        val listenerset: MutableSet<RequestListener> = HashSet()
        listenerset.add(FrescoTraceListener())
        val config = ImagePipelineConfig.newBuilder(mContext).setRequestListeners(listenerset)
            .build()
        Fresco.initialize(mContext, config)
    }
}